HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -205,6 +205,13 @@ def self.apply_header(hpxml, epw_file)
       hpxml.header.natvent_days_per_week = 3
       hpxml.header.natvent_days_per_week_isdefaulted = true
     end
+
+    hpxml.header.vacancy_periods.each do |vacancy_period|
+      if vacancy_period.natvent.nil?
+        vacancy_period.natvent = false
+        vacancy_period.natvent_isdefaulted = true
+      end
+    end
```

Comment:
This should be power outages instead, no?

Reply:
It's intentionally vacancy period here. When vacant, we assume natural ventilation is always unavailable.